### ISSUE TITLE
How to avoid the extra `span`?

### ISSUE THREAD

**Issue by freewind** [widok/widok#51]
```scala
val names :Var[Seq[String]] = Var(Seq("Mike", "Jeff"))

override def view(): View = {
    div(
      names.map(_.map(div(_))).map(div(_))
    )
}
```

Will generate: 

```html
<span><div><div>Mike</div><div>Jeff</div></div></span>
```

You can see there is an extra `span` at the outest level. Is there anyway to avoid it?

Since the channels heavily used in my code, there will be a lot of un-needed `span` which make the styling and reading a little bit harder.

**Comment by tindzk** [comment]
Thanks for reporting. It was fixed in e697f03.

**Comment by freewind** [comment]
Thanks for fixing it so fast. I will try it tomorrow :+1:
